backend/corpora/common/upload.py

```diff
@@ -79,6 +79,7 @@ def upload(
         if dataset.processing_status.processing_status not in [
             ProcessingStatus.SUCCESS,
             ProcessingStatus.FAILURE,
+            ProcessingStatus.INITIALIZED,
```

Comment:
Will there not be corresponding test(s) that need to be updated?

Reply:
no because we don't have tests for this

Reply:
getting lambdas working in localstack has proven difficult. Andrew dove into this and failed, I think. 😬